install.php

```diff
@@ -40,6 +40,20 @@ class Com_RedcoreInstallerScript
 	public $installer = null;
 
 	/**
+	 * Extension element name
+	 *
+	 * @var  string
+	 */
+	protected $extensionElement = '';
+
+	/**
+	 * Old version according to manifest
+	 *
+	 * @var  string
+	 */
+	protected $oldVersion = '0.0.0';
```

Comment:
I used this logic https://github.com/joomla/joomla-cms/blob/staging/libraries/cms/installer/installer.php#L1101-L1104